modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,115 @@
+= Modify API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your business requirement changes, you might want to modify an API Group by adding new API instances to it. To cover new scenarios, you might want to add a new API instance. 
+
+You can also version an existing instance to add customizations. If a service is no longer required, you can remove the respective API instance that supports the service.
+
+The following tasks are covered in this section:
+
+* <<adding-api-instances-to-groups, Add API instances to Existing API Groups>>
+* <<removing-api-instances-from-groups, Remove API instances from an API Group>>
+* <<creating-new-versions-of-groups, Create new versions of an API Group>>
+
+
+== Restrictions for Modifying API Group and Instances
+
+Before you modify an API Group to add or remove API Group instances, ensure that you carefully reviews the restrictions. You can modify the API Group instance anytime if you meet these restrictions. You can:
+
+* Add new API instances of APIs that already exist in the group only if the API Group instance has no contracts. 
+* Add only those API instances that belong to the same identity provider (IdP) as the API Group instance. You cannot mix IdPs in an API Group instance.
+* Add API instances but not APIs if:
+** The API Group instance is published.
+** The API Group version is published.
+** There are more than one API Group instances of the same API Group version.
+
+== Exchange Status Based on API Group Changes
+
+The status of the API Group in Exchange changes Based on the scenario:
+
+* If your API Group is published in Exchange, your Exchange asset reflects the  new API instance. 
+* If the API instance you added is related to an asset that is already linked to the Exchange API Groups asset, the the API instance is reflected.
+* If the API instance you added belongs to a new asset that is not yet liked to the Exchange API Groups asset, the consumers are required to create a new version of the API Groups.
```

Comment:
"the consumers": GLOBALLY "your customers"? "your users"?
```suggestion
* If the API instance you added belongs to a new asset that is not yet liked to the Exchange API Groups asset, the consumers must create a new version of the API Groups asset.
```
Why use "asset"?

Reply:
Done! Thanks!